kits/google-reviews-ai-manager/.env.example

```
@@ -0,0 +1,10 @@
+GOOGLE_CLIENT_ID="your_google_client_id"
+GOOGLE_CLIENT_SECRET="your_google_client_secret"
+NEXTAUTH_SECRET="your_nextauth_secret"
+NEXTAUTH_URL="http://localhost:3000"
+
+# Lamatic API Keys
+LAMATIC_API_KEY="your_lamatic_api_key"
+LAMATIC_PROJECT_ID="your_lamatic_project_id"
+LAMATIC_API_URL="your_lamatic_api_url"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)